src/NuGet.Indexing/Sql2Lucene.cs

```diff
@@ -280,5 +284,30 @@ public static void Export(string sourceConnectionString, string destinationPath,
 
             stopwatch.Reset();
         }
+
+        private static DateTime GetCommitTimestampFromCatalog(Uri indexUrl, ILogger<Sql2Lucene> logger)
+        {
+            DateTime commitTime = DateTime.UtcNow;
+            try
+            {
+                using (var client = new System.Net.Http.HttpClient())
+                {
+                    var response = client.GetAsync(indexUrl).Result;
+                    logger.LogInformation("Fetching catalog index page: {0}", response.StatusCode);
+                    response.EnsureSuccessStatusCode();
+
+                    string json = response.Content.ReadAsStringAsync().Result;
+                    JObject obj = JObject.Parse(json);
+                    commitTime = obj["commitTimeStamp"].ToObject<DateTime>();
```

Comment:
It is necessary. This time is passed to `DocumentCreator` which expects a `DateTime` object.